pkg/mounter/proxy/server/metrics.go

```diff
@@ -0,0 +1,328 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/kubernetes-sigs/alibaba-cloud-csi-driver/pkg/utils"
+	"k8s.io/klog/v2"
+	"k8s.io/mount-utils"
+	mountutils "k8s.io/mount-utils"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	MetricsMountRetryCount          = "mount_retry_count"
+	MetricsMountPointStatus         = "mount_point_status"
+	MetricsMountPointFailoverCount  = "mount_point_failover_count"
+	MetricsLastFuseClientExitReason = "last_fuse_client_exit_reason"
+)
+
+var maxCountRecord int = 999
+
+// MonitorState represents the state of a monitor
+type MonitorState int
+
+const (
+	MonitorStateInitialized MonitorState = iota
```

Comment:
I think we lack the necessary monitor state here. At least we also have a terminating monitor state.